Remove (or suppress if needed) the build warnings for cleaner logs

See workflow [run](https://github.com/GlobalDevOpsBootcamp/eShopOnWeb/runs/4902981020?check_suite_focus=true) for an example. Lots of warnings on the same thing in the `Build with dotnet` step.

```
Warning: /home/runner/work/eShopOnWeb/eShopOnWeb/tests/FunctionalTests/PublicApi/ApiTestFixture.cs(72,41): warning CA2017: Number of parameters supplied in the logging message template do not match the number of named placeholders 
```

--- a/tests/FunctionalTests/PublicApi/ApiTestFixture.cs
+++ b/tests/FunctionalTests/PublicApi/ApiTestFixture.cs
@@ -21,14 +21,14 @@ public class ApiTestFixture : WebApplicationFactory<Startup>
         {
             services.AddEntityFrameworkInMemoryDatabase();
 
-                // Create a new service provider.
-                var provider = services
-                .AddEntityFrameworkInMemoryDatabase()
-                .BuildServiceProvider();
+            // Create a new service provider.
+            var provider = services
+            .AddEntityFrameworkInMemoryDatabase()
+            .BuildServiceProvider();
 
-                // Add a database context (ApplicationDbContext) using an in-memory 
-                // database for testing.
-                services.AddDbContext<CatalogContext>(options =>
+            // Add a database context (ApplicationDbContext) using an in-memory 
+            // database for testing.
+            services.AddDbContext<CatalogContext>(options =>
             {
                 options.UseInMemoryDatabase("InMemoryDbForTesting");
                 options.UseInternalServiceProvider(provider);
@@ -40,12 +40,12 @@ public class ApiTestFixture : WebApplicationFactory<Startup>
                 options.UseInternalServiceProvider(provider);
             });
 
-                // Build the service provider.
-                var sp = services.BuildServiceProvider();
+            // Build the service provider.
+            var sp = services.BuildServiceProvider();
 
-                // Create a scope to obtain a reference to the database
-                // context (ApplicationDbContext).
-                using (var scope = sp.CreateScope())
+            // Create a scope to obtain a reference to the database
+            // context (ApplicationDbContext).
+            using (var scope = sp.CreateScope())
             {
                 var scopedServices = scope.ServiceProvider;
                 var db = scopedServices.GetRequiredService<CatalogContext>();
@@ -54,23 +54,23 @@ public class ApiTestFixture : WebApplicationFactory<Startup>
                 var logger = scopedServices
                     .GetRequiredService<ILogger<ApiTestFixture>>();
 
-                    // Ensure the database is created.
-                    db.Database.EnsureCreated();
+                // Ensure the database is created.
+                db.Database.EnsureCreated();
 
                 try
                 {
-                        // Seed the database with test data.
-                        CatalogContextSeed.SeedAsync(db, loggerFactory).Wait();
+                    // Seed the database with test data.
+                    CatalogContextSeed.SeedAsync(db, loggerFactory).Wait();
 
-                        // seed sample user data
-                        var userManager = scopedServices.GetRequiredService<UserManager<ApplicationUser>>();
+                    // seed sample user data
+                    var userManager = scopedServices.GetRequiredService<UserManager<ApplicationUser>>();
                     var roleManager = scopedServices.GetRequiredService<RoleManager<IdentityRole>>();
                     AppIdentityDbContextSeed.SeedAsync(userManager, roleManager).Wait();
                 }
                 catch (Exception ex)
                 {
-                    logger.LogError(ex, $"An error occurred seeding the " +
-                        "database with test messages. Error: {ex.Message}");
+                    logger.LogError(ex, "An error occurred seeding the " +
+                        $"database with test messages. Error: {ex.Message}");
                 }
             }
         });

--- a/tests/FunctionalTests/PublicApi/ApiTestFixture.cs
+++ b/tests/FunctionalTests/PublicApi/ApiTestFixture.cs
@@ -19,8 +19,6 @@ public class ApiTestFixture : WebApplicationFactory<Startup>
 
         builder.ConfigureServices(services =>
         {
-            services.AddEntityFrameworkInMemoryDatabase();
-
             // Create a new service provider.
             var provider = services
             .AddEntityFrameworkInMemoryDatabase()

--- a/tests/FunctionalTests/Web/WebTestFixture.cs
+++ b/tests/FunctionalTests/Web/WebTestFixture.cs
@@ -21,14 +21,14 @@ public class WebTestFixture : WebApplicationFactory<Startup>
         {
             services.AddEntityFrameworkInMemoryDatabase();
 
-                // Create a new service provider.
-                var provider = services
-                .AddEntityFrameworkInMemoryDatabase()
-                .BuildServiceProvider();
+            // Create a new service provider.
+            var provider = services
+                            .AddEntityFrameworkInMemoryDatabase()
+                            .BuildServiceProvider();
 
-                // Add a database context (ApplicationDbContext) using an in-memory 
-                // database for testing.
-                services.AddDbContext<CatalogContext>(options =>
+            // Add a database context (ApplicationDbContext) using an in-memory 
+            // database for testing.
+            services.AddDbContext<CatalogContext>(options =>
             {
                 options.UseInMemoryDatabase("InMemoryDbForTesting");
                 options.UseInternalServiceProvider(provider);
@@ -40,12 +40,12 @@ public class WebTestFixture : WebApplicationFactory<Startup>
                 options.UseInternalServiceProvider(provider);
             });
 
-                // Build the service provider.
-                var sp = services.BuildServiceProvider();
+            // Build the service provider.
+            var sp = services.BuildServiceProvider();
 
-                // Create a scope to obtain a reference to the database
-                // context (ApplicationDbContext).
-                using (var scope = sp.CreateScope())
+            // Create a scope to obtain a reference to the database
+            // context (ApplicationDbContext).
+            using (var scope = sp.CreateScope())
             {
                 var scopedServices = scope.ServiceProvider;
                 var db = scopedServices.GetRequiredService<CatalogContext>();
@@ -54,23 +54,23 @@ public class WebTestFixture : WebApplicationFactory<Startup>
                 var logger = scopedServices
                     .GetRequiredService<ILogger<WebTestFixture>>();
 
-                    // Ensure the database is created.
-                    db.Database.EnsureCreated();
+                // Ensure the database is created.
+                db.Database.EnsureCreated();
 
                 try
                 {
-                        // Seed the database with test data.
-                        CatalogContextSeed.SeedAsync(db, loggerFactory).Wait();
+                    // Seed the database with test data.
+                    CatalogContextSeed.SeedAsync(db, loggerFactory).Wait();
 
-                        // seed sample user data
-                        var userManager = scopedServices.GetRequiredService<UserManager<ApplicationUser>>();
+                    // seed sample user data
+                    var userManager = scopedServices.GetRequiredService<UserManager<ApplicationUser>>();
                     var roleManager = scopedServices.GetRequiredService<RoleManager<IdentityRole>>();
                     AppIdentityDbContextSeed.SeedAsync(userManager, roleManager).Wait();
                 }
                 catch (Exception ex)
                 {
-                    logger.LogError(ex, $"An error occurred seeding the " +
-                        "database with test messages. Error: {ex.Message}");
+                    logger.LogError(ex, "An error occurred seeding the " +
+                        $"database with test messages. Error: {ex.Message}");
                 }
             }
         });

--- a/tests/FunctionalTests/Web/WebTestFixture.cs
+++ b/tests/FunctionalTests/Web/WebTestFixture.cs
@@ -19,8 +19,6 @@ public class WebTestFixture : WebApplicationFactory<Startup>
 
         builder.ConfigureServices(services =>
         {
-            services.AddEntityFrameworkInMemoryDatabase();
-
             // Create a new service provider.
             var provider = services
                             .AddEntityFrameworkInMemoryDatabase()

